<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebServices</class>
 <widget class="QDialog" name="WebServices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Web service connection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/subsurface-icon</normalon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="userID">
     <property name="placeholderText">
      <string>Enter your ID here</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>420</width>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="download">
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>User ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="status">
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QCheckBox" name="saveUidLocal">
     <property name="text">
      <string>Save user ID locally?</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passLabel">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="password">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="upload">
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>userID</tabstop>
  <tabstop>password</tabstop>
  <tabstop>download</tabstop>
  <tabstop>upload</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../subsurface.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WebServices</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WebServices</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
